### PR TITLE
8 key takeaways from Tim Glaser, CTO of PostHog

### PR DESCRIPTION
**Pull request by cosydney**

### CODE REVIEW

**Comment by cosydney** [comment]
TODO: add video !

**Comment by cosydney** [comment]
[Slack File: CleanShot 2023-01-23 at 16.24.49@2x.jpg](https://files.slack.com/files-pri/T016LMS8GQN-F04KXASCY86/cleanshot_2023-01-23_at_16.24.49_2x.jpg)
[Slack File: CleanShot 2023-01-23 at 16.24.19@2x.jpg](https://files.slack.com/files-pri/T016LMS8GQN-F04LGHKLWCR/cleanshot_2023-01-23_at_16.24.19_2x.jpg)

**Review by acoudouy — APPROVED** [review]
looks good!

File: data/p/8-key-takeaways-from-tim-glaser-cto-of-postHog.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: "8 key takeaways from Tim Glaser, CTO of PostHog"
+date: "2023-01-23"
+tags: ["software development"]
+draft: false
+summary: "8 key takeaways from CTO of PostHog: Build on OSS, prioritize code quality, ignore security extortionists, get a coach, invest in tools, make people pay for your product, use favorite tools, have a bigger vision for projects. #CTO #opensource #productdevelopment"
```
Comment (acoudouy) [inline data/p/8-key-takeaways-from-tim-glaser-cto-of-postHog.mdx:6]: les # ça me fait bizarre perso ^^

Comment (cosydney) [reply data/p/8-key-takeaways-from-tim-glaser-cto-of-postHog.mdx:6]: c'est chatgpt je les enleves

**Pull request merged**